contracts/src/v0.8/dev/interfaces/OCR2DROracleInterface.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+/**
+ * @title OCR2DR oracle interface.
+ */
+interface OCR2DROracleInterface {
+  function sendRequest(
+    address sender,
+    uint256 nonce,
+    uint256 subscriptionId,
```

Comment:
I am assuming the subscriptionId is for billing purposes, but what is the nonce for?

Reply:
nonce if for Oracle to differentiate client requests.